flake.lock

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
my concern or misunderstanding is now this duplicates the niv entries ?
is there a way for flake to use Niv so we we can homogenize the two ?